### ISSUE TITLE
[BUG v. 0.6.x] NgCalendarModule not found in Angular 9, Ionic 5 and calendar v. 0.6.x

### ISSUE THREAD

**Issue by ilprima** [twinssbc/Ionic2-Calendar#546]
Using the calendar version 0.6.x it is not possible to import NgCalendarModule. To reproduce this problem just create a new project with Ionic 5 and Angular 9. You can see that NgCalendarModule does not exist in ionic2-calendar, it means import {NgCalendarModule} from 'ionic2-calendar' it is not importable saying to add @NgModule in ionic2-calendar file.

I think also that using v. 0.5.x with Ionic 5 and Angular 9 breaks Ivy compilation

**Comment by SansDK** [comment]
Working here.

I have this in my module file:
```
import { NgCalendarModule } from 'ionic2-calendar';

@NgModule({
  imports: [
    NgCalendarModule,
    ...
  ],

...
```

Package versions:
ionic2-calendar v0.6.4
Angular v9.1.9
Ionic v5.26.0

**Comment by craigsh** [comment]
I have a similar issue. The calendar works, but in my template it shows as an error - hovering over it shows 'calendar' is not a known element.

I have imported NgCalendarModule as shown above.

For me, it's just an annoyance.

![image](https://user-images.githubusercontent.com/12677787/83688740-fb742780-a641-11ea-8143-b4fdb227e8d7.png)

**Comment by SansDK** [comment]
@craigsh I have the same issue.

Please comment on issue #545 and / or +1 it.

**Comment by SatyaShodhaka** [comment]
@craigsh Hey could you paste the solution in here when you find it pls

**Comment by twinssbc** [comment]
could you try the latest version 0.6.5?

**Comment by twinssbc** [comment]
Should be fixed in version 0.6.5
